pkgs/applications/networking/cluster/terraform-docs/default.nix

```diff
@@ -1,21 +1,21 @@
 { lib, buildGoModule, fetchFromGitHub }:
 buildGoModule rec {
   pname = "terraform-docs";
-  version = "0.13.0";
+  version = "0.14.1";
 
   src = fetchFromGitHub {
     owner = "terraform-docs";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-lx+yp0ybgZfmxvPM2BY2yq39qA6XebcKGrFd0AJa4yg=";
+    sha256 = "sha256-Jm0ySxn4GFW4iAH3tOIvclcDGJMKzH7m7fhWnAf4+gs=";
   };
 
-  vendorSha256 = "sha256-qoZUgSSr7jsDVVPBUyfN5Uw4CnH9EnD/4tX+TCSNV0Q=";
+  vendorSha256 = "sha256-IzmAlthE6SVvGHj72wrY1/KLehOv8Ck9VaTv5jMpt48=";
 
   subPackages = [ "." ];
 
   preBuild = ''
-    buildFlagsArray+=("-ldflags" "-X main.version=${version}")
+    buildFlagsArray+=("-ldflags" "-X github.com/terraform-docs/terraform-docs/internal/version.commit=${version}")
```

Comment:
```
$ ./results/terraform-docs/bin/terraform-docs --version
terraform-docs version v0.14.1 0.14.1 linux/amd64
```
I think the commit is meant to contain the git commit ID. Since we are using the tagged release it's probably fine to remove it entirely and just have the version in there. TBI

Reply:
Fair call!